test/generator_test.exs

```diff
@@ -230,7 +256,31 @@ defmodule BlueBird.Test.GeneratorTest do
 
       assert request.query_params == %{"s" => "poodle"}
       assert request.path_params == %{"id" => "137"}
-      assert request.body_params == %{"betty" => "white"}
+      assert request.body_params == %{
```

Comment:
What do we gain from this? The generator only sets `request.body_params` to `conn.body_params`. So we only have to test whether the field was set as expected, but we don't have to test whether Plug is able to decode complex json objects.

Reply:
Mhh, you might be right... this test is maybe not that useful. In fact the function itself is tested and it is not necessary anymore to do that here. I will change that.